python/paddle/audio/datasets/esc50.py

```diff
@@ -167,14 +193,16 @@ def __init__(
             files=files, labels=labels, feat_type=feat_type, **kwargs
         )
 
-    def _get_meta_info(self) -> list[collections.namedtuple]:
+    def _get_meta_info(self) -> list[MetaInfoState]:
```

Comment:
```suggestion
    def _get_meta_info(self) -> list[meta_info]:
```

直接用 `meta_info` 就行 ～

Reply:
收到